feature/profile/src/main/java/com/yapp/feature/profile/ProfileScreen.kt

```diff
@@ -109,18 +110,18 @@ private fun ProfileScreen(
                     modifier = Modifier
                         .padding(horizontal = 20.dp)
                         .height(56.dp),
-                    onClickSettings = onClickSettings
+                    onClickSettings = { onIntent(ProfileIntent.ClickLogout) }
```

Comment:
```suggestion
                    onClickSettings = { onIntent(ProfileIntent.ClickSettings) }
```
R: ClickSettings 호출하도록 수정해주세요

Reply:
[c3a484d](https://github.com/YAPP-admin/yappu-world-android/pull/141/commits/c3a484d0ffa3d174d22c01578f13af903a6af0f6)

확인을 제대로 안했네요 ;; 수정처리했습니다